manifests/0000_10_config-operator_01_consoleclidownload.crd.yaml

```diff
@@ -0,0 +1,295 @@
+                      otherwise 422 (Unprocessable Entity) will be returned.
+                    type: boolean
+                  controller:
+                    description: If true, this reference points to the managing controller.
+                    type: boolean
+                  kind:
+                    description: 'Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+                    type: string
+                  name:
+                    description: 'Name of the referent. More info: http://kubernetes.io/docs/user-guide/identifiers#names'
+                    type: string
+                  uid:
+                    description: 'UID of the referent. More info: http://kubernetes.io/docs/user-guide/identifiers#uids'
+                    type: string
+                required:
+                - apiVersion
+                - kind
+                - name
+                - uid
+                type: object
+              type: array
+            resourceVersion:
+              description: "An opaque value that represents the internal version of
+                this object that can be used by clients to determine when objects
+                have changed. May be used for optimistic concurrency, change detection,
+                and the watch operation on a resource or set of resources. Clients
+                must treat these values as opaque and passed unmodified back to the
+                server. They may only be valid for a particular resource or set of
+                resources. \n Populated by the system. Read-only. Value must be treated
+                as opaque by clients and . More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency"
+              type: string
+            selfLink:
+              description: "SelfLink is a URL representing this object. Populated
+                by the system. Read-only. \n DEPRECATED Kubernetes will stop propagating
+                this field in 1.20 release and the field is planned to be removed
+                in 1.21 release."
+              type: string
+            uid:
+              description: "UID is the unique in time and space value for this object.
+                It is typically generated by the server on successful creation of
+                a resource and is not allowed to change on PUT operations. \n Populated
+                by the system. Read-only. More info: http://kubernetes.io/docs/user-guide/identifiers#uids"
+              type: string
+          type: object
+        spec:
+          description: ConsoleCLIDownloadSpec is the desired cli download configuration.
+          properties:
+            description:
+              description: description is the description of the CLI download (can
+                include markdown).
+              type: string
+            displayName:
```

Comment:
can this be empty?

Reply:
It's a required field, so it should not be empty.

Reply:
Are we allowed to change this now?  API has been in for a release already.